### PR TITLE
A2

### PR DESCRIPTION
**Pull request by partamonov**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -61,6 +61,7 @@ type Client struct {
 	NetworkLists *NetworkListService
 	PropertyAPI  *PropertyAPIService
 	ReportingAPI *ReportingAPIService
+	A2API        *AdaptiveAccelerationAPIService
```
Comment (RafPe) [inline client.go]: Maybe we can skip naming it `API` ?

File: client.go [hunk 2]
```diff
@@ -85,6 +86,7 @@ var (
 		"network_list": "/network-list/v1/network_lists",
 		"papi_v1":      "/papi/v1",
 		"reporting_v1": "/reporting-api/v1/reports",
+		"a2_v1":        "/adaptive-acceleration/v1/properties",
```
Comment (RafPe) [inline client.go]: Maybe we should use the same approach as with const type for env variables ?

File: client.go (hunk 1 above)
Comment (partamonov) [reply client.go]: Good one

File: client.go (hunk 2 above)
Comment (partamonov) [reply client.go]: I can agree that having it in constants would be better. I will move only A2 one now. All others let's do via different pull request, as well as API removal from comment above

**Pull request merged**